scripts/webext-test-functional

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+export MOCHA_TIMEOUT=60000
```

Comment:
Nit, we could also restrict the environment variables to the web-ext functional tests by doing something like the following at line 22: (line 20 in the original version of this bash script):

```
MOCHA_TIMEOUT=60000 npm run test-functional
```

Reply:
e3a849c9